haystack/components/evaluators/document_map.py

```diff
@@ -6,19 +6,19 @@
 @component
 class DocumentMeanAveragePrecision:
     """
-    Evaluator that calculates the mean average precision of the retrieved documents.
+    Evaluator that calculates the mean average precision of the retrieved documents, a metric
+    that measures how high retrieved documents are ranked.
     Each question can have multiple ground truth documents and multiple retrieved documents.
 
+    `DocumentMeanAveragePrecision` doesn't normalize its inputs, the `DocumentCleaner` component
+    can be used to clean the documents before passing them to this evaluator.
```

Comment:
```suggestion
    should be used to clean and normalize the documents before passing them to this evaluator.
```
Since we are comparing strings.